client/dive-common/components/RunPipelineMenu.vue

```diff
@@ -149,32 +152,34 @@ export default defineComponent({
             <v-col
               v-for="(pipeType) in Object.keys(pipelines)"
               :key="pipeType"
-              cols="6"
+              cols="12"
             >
               <v-menu
                 :key="pipeType"
                 offset-x
-                right
+                nudge-left="10"
+                left
               >
                 <template v-slot:activator="{ on }">
                   <v-btn
                     depressed
                     block
                     v-on="on"
                   >
-                    {{ pipeTypeDisplay(pipeType) }}
                     <v-icon
-                      left
+                      right
                       color="accent"
-                      class="ml-0"
+                      class="mr-0"
                     >
-                      mdi-menu-right
+                      mdi-menu-left
```

Comment:
I'm guessing it's mostly from only knowing right-to-left languages but it feels different. This isn't me requesting a change or anything and I'd wait to see what others say.  I played around for a bit seeing if I could do some stuff width nudging and moving it around but nothing was better than what you currently have.

Reply:
Yeah, it's  odd.  If you want to make them open to the right and just not worry about the case where it covers the lower menu, I'm good with that.

Reply:
That sounds good.